### PR TITLE
Configuration exceptions exception cases on master.

### PR DESCRIPTION
**Pull request by SpacePossum**
replaces; 
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1500

### CODE REVIEW

File: Symfony/CS/Fixer/Symfony/ExtraEmptyLinesFixer.php [hunk 1]
```diff
@@ -43,6 +44,8 @@
      * - 'use' remove blank lines between 'use' import statements
      *
      * @param string[]|null $configuration
+     *
+     * @throws InvalidFixerConfigurationException
```
Comment (keradus) [inline Symfony/CS/Fixer/Symfony/ExtraEmptyLinesFixer.php]: We don't use that annotation anywhere.
Introduce it everywhere or nowhere. Use them in 1% of places is just a noise.

**Comment by keradus** [comment]
No changes needed inside configuration resolver?

**Comment by keradus** [comment]
Please, could you fix the build ?

**Comment by SpacePossum** [comment]
tests should be OK now, but lets wait a bit longer for all to finish ;)

**Comment by keradus** [comment]
:+1:

**Comment by gharlan** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**